### DISCUSSION TITLE
Paramétrage des plages de visualisation

### DISCUSSION THREAD

**Discussion by IGNF-Xavier** [IGNF-Ma-carte/.github#2] [Visualiseur] [answered]
La DREAL Normandie ( identifiant github : jpotel) a testé l'intégration des flux WFS et WMS configurés au travers du swagger de l'API Entrepôt.

L'utilisateur a trouvé complexe le paramètrage par niveau de zoom et pas par échell

**Comment by Viglino** [comment, marked as answer]
La notion d'échelle a peut de pertinence dans notre cas. La projection utilisée (Mercator) ne conservant pas les distance, l'échelle n'est pas la même un sud et au nord de la France à même niveau de zoom.
Classiquement, les web services résonnent en niveau de zoom (WFS, WMS)...

**Reply by jpotel** [reply]
Oui comme pour le RGF93/Lambert93 mais cela ne nous empêche pas de produire des cartes avec une échelle ! 
Pour l'être humain le niveau de zoom ne veut rien dire du tout. Je sais ce que signifie une échelle de 1/25000e que ce soit dans le nord de la France et en Guyane mais je ne sais pas à quoi correspond le niveau de zoom 12 en Normandie. Je suppose qu'il existe un moyen mathématique pour donner une estimation de l'échelle en un point précis (exemple : centre de la l'étendu max de la carte).

**Reply by Viglino** [reply]
La solution serait peut-être de préciser le niveau de zoom de la carte au moment de l'ouverture du dialogue.
